[BUG] Build error when using `make apalache-jar`

<!-- Thank you for filing a report! Please ensure you have filled out all -->
<!-- sections, as it help us to address the problem effectively. -->

## Description

Building Apalache using `make apalache-jar`, as described in the [documentation](https://apalache.informal.systems/docs/apalache/installation/source.html), leads to the following error:
```
[ERROR] Failed to execute goal on project tla-pp: Could not resolve dependencies for project at.forsyte.apalache:tla-pp:jar:0.19.2-SNAPSHOT: Could not find artifact at.forsyte.apalache:tlair:jar:tests:0.19.2-SNAPSHOT in oss-sonatype (https://oss.sonatype.org/content/repositories/snapshots/) -> [Help 1]
```
Building with `make` works without problems. The problem with `make apalache-jar` happens after running both `mvn clean && mvn package` and `make`.

## Expected behavior

No error.

## System information

- Apalache version [`apalache-mc version`]: v0.19.2
- OS [e.g. Ubuntu Linux or Mac OS, and the current version]: Linux Mint 20.2
- JDK version [e.g. OpenJDK 0.8.0]: openjdk 11.0.13

--- a/tla-pp/pom.xml
+++ b/tla-pp/pom.xml
@@ -14,6 +14,28 @@
     <name>tla-pp</name>
     <url>https://github.com/informalsystems/apalache</url>
 
+    <profiles>
+        <!-- Workaround to allow a test-only internal dependency without breaking maven.test.skip -->
+        <!-- See https://issues.apache.org/jira/browse/MJAR-138?focusedCommentId=16160394&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16160394 -->
+        <profile>
+            <activation>
+                <property>
+                    <name>maven.test.skip</name>
+                    <value>!true</value>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>at.forsyte.apalache</groupId>
+                    <artifactId>tlair</artifactId>
+                    <version>${project.version}</version>
+                    <type>test-jar</type>
+                    <scope>test</scope>
+                </dependency>
+            </dependencies>
+        </profile>
+    </profiles>
+
     <dependencies>
         <dependency>
             <groupId>at.forsyte.apalache</groupId>
@@ -63,13 +85,6 @@
         <dependency>
             <groupId>org.scalacheck</groupId>
             <artifactId>scalacheck_${scalaBinaryVersion}</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>at.forsyte.apalache</groupId>
-            <artifactId>tlair</artifactId>
-            <version>${project.version}</version>
-            <type>test-jar</type>
             <scope>test</scope>
         </dependency>
     </dependencies>

